src/components/Person.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import './Person.scss';
+import PropTypes from 'prop-types';
+
+const partnerSex = sex => (
+  sex === 'm'
+    ? 'wife'
+    : 'husband'
+);
+
+export const Person = ({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) => (
+  <>
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+      <p className="Person__age">
+        {age
+          ? `I am ${age}`
+          : ''
+        }
+      </p>
```

Comment:
use conditional rendering with && or || (you no need to render this `p` at all if there is no age)